README.md

```diff
@@ -77,6 +77,34 @@ var updateSourceData = function(source) {
 videojs.Html5DashJS.hook('updatesource', updateSourceData);
 ```
 
+## Captions
+
+As of `video.js@5.14`, native captions are no longer supported on any browser besides Safari. Dash can handle captions referenced embedded vtt files, embedded captions in the manifest, and with fragmented text streaming. It is impossible to use video.js captions when dash.js is using fragmented text captions, so the user must disable native captions when using `videojs-contrib-dash`.
+
+```javascript
+videojs('example-video', {
+  html5: {
+    nativeCaptions: false,
+  },
+});
+```
+
+A warning will be logged if this setting is not applied.
+
+### Automatic Track Selection
+
+`videojs-contrib-dash` will listen for `video.js`'s built-in text track change events and will update the dash.js text track automatically.
+
+If the user wants `videojs-contrib-dash` to load a list of the text tracks but then to allow the user to programatically respond to `video.js`'s built-in text track change events, then pass the `automaticTextTrackSelection` option:
+
+```javascript
+videojs('example-video', {
+  html5: {
+    automaticTextTrackSelection: false,
```

Comment:
I opted to _not_ add it to `html5.dash` because that's where the options for `dashjs` are stored. This is what I was referencing in https://github.com/videojs/videojs-contrib-dash/pull/144#issuecomment-277023964 when I suggested it might be beneficial to have the options passed to `dashjs` in a separate object like `html5.dash.dashjsOptions`.

https://github.com/videojs/videojs-contrib-dash/pull/135/files#diff-a10f3c20a3b12e434e950fa44ed404d2R49

https://github.com/videojs/videojs-contrib-dash/pull/135/files#diff-a10f3c20a3b12e434e950fa44ed404d2R49